### ISSUE TITLE
HTTP redirects in rdf-load step

### ISSUE THREAD

**Issue by timathom** [ndw/xmlcalabash1-rdf#1]
This is my first experience with XProc, and I've found these extension steps to be extremely useful--thank you! 

I'd like to submit a feature request (if I may) for handling HTTP redirects in the `rdf-load` step. For example, the following URI supports content negotiation with a 303 redirect response depending on the content type requested: http://www.worldcat.org/oclc/892894708. It would be helpful to have something like a `follow-redirect` option on `rdf-load` to allow for greater flexibility when requesting data from different sources (for example, I'd like to be able to change the `language` option from an XForms interface without having to modify the file extension).

**Comment by ndw** [comment]
It's a good idea. I probably need to integrate a better HTTP client into this library.

How does follow-redirect come into play with respect to selecting the language?

**Comment by timathom** [comment]
Yes, being able to control HTTP headers from this library would make a big difference (e.g., supporting content negotiation via the language option).
